bin/faucet/frontend/app.js

```diff
@@ -192,6 +200,54 @@ export class MidenFaucetApp {
         }
     }
 
+    pollNote(noteId) {
+        return new Promise((resolve, reject) => {
+            // Poll every 500ms for the first 10 seconds, then every 1s for the next 30 seconds, then every 5s.
+            let currentInterval = 500;
+            let pollInterval;
```

Comment:
nit: prefer variable initialization
```suggestion
            let pollInterval = 0;
```